### DISCUSSION TITLE
Custom placeholders (for PO files)

### DISCUSSION THREAD

**Discussion by silopolis** [WeblateOrg/weblate#9113] [Q&A]
Hi,

Discussing with the good folks from po4a project [here](https://github.com/mquinson/po4a/issues/400#issuecomment-1509952186) we came to the question of defining custom placeholders and their possible support in TMSs…

After making due searches, I can't make myself a clear idea about Weblate possibilities on this matter. Could you please shed some light on this for me ?

TY
J

**Comment by nijel** [comment]
Fully custom placeholders can be defined using https://docs.weblate.org/en/latest/user/checks.html#check-placeholders, but you're better to stick with some of the standard formatting options like https://docs.weblate.org/en/latest/user/checks.html#formatted-strings
